src/app/components/resource/resource.component.html

```diff
@@ -7,20 +7,29 @@
     <span class="h4 mb-0">
       {{ resource.name }}
     </span>
-    <div class="btn-group" role="group">
+    <div class="btn-group justify-content-end flex-wrap" role="group">
       <button
         (click)="openUpdateModal(resource); $event.stopPropagation()"
-        class="btn btn-warning"
+        class="btn btn-warning mb-2 mb-sm-2 mb-lg-0"
       >
         <i class="fa fa-pencil"></i> Details
       </button>
+
       <button
         (click)="openGenerateModal(resource); $event.stopPropagation()"
-        class="btn btn-secondary"
+        class="btn btn-secondary mb-2 mb-sm-2 mb-lg-0"
         *ngIf="!isMetaTool()"
       >
         <i class="fa fa-plus"></i> Generate multiple drafts
       </button>
+      <button
+        (click)="
+          openAttributeTranslationsModal(resource); $event.stopPropagation()
+        "
+        class="btn btn-primary"
+      >
+        <i class="fa fa-language"></i> Translated Attributes
+      </button>
```

Comment:
we should make this button only visible for non-meta tools.

```
*ngIf="!isMetalTool()"
```

Reply:
Also, let's move this button between the "Details" and "Generate multiple drafts" buttons

Reply:
Added `*ngIf="!isMetalTool()"` onto button and moved between "Details" and "Generate multiple drafts".